default_metadata/component/autocomplete.json

```diff
@@ -0,0 +1,25 @@
+{
+  "_id": "component.autocomplete",
+  "_type": "autocomplete",
+  "errors": {},
+  "hint": "",
+  "options": [
+    {
+      "_id": "autocomplete_option_1",
+      "_type": "select_option",
+      "label": "Option",
+      "value": "value-1"
+    },
+    {
+      "_id": "autocomplete_option_2",
+      "_type": "select_option",
+      "label": "Option",
+      "value": "value-2"
+    }
+  ],
+  "name": "component-name",
+  "legend": "Question",
+  "validation": {
+    "required": true
+  }
+}
```

Comment:
i don't think we need options to be prefilled here. these are dynamic that are only loaded by the frontend, otherwise it's just a select list